### PR TITLE
changes

### PR DESCRIPTION
**Pull request by RANJEETJ06**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Review by siriak — COMMENTED** [review]
What's the purpose of the change? Only to add braces? It seems that there are no other changes

**Comment by RANJEETJ06** [comment]
Changes:
All the common logic parts like node->node.next  in both if and else gets combined along correction of some non-code errors like /** changes to /*
the main changes are like 
while(){
if(something){
i++;
node=node.next;
else{
i--;
node=node.next
}

i change it to like 
while(){
if(){
i++}
else{
i--}
node=node.next
}

**Comment by albina-astr** [comment]
Hi. Please add some clear commit message and describe your changes shortly. It would be much easier to understand the code changes.

File: src/main/java/com/thealgorithms/datastructures/trees/BinaryTree.java [hunk 1]
```diff
@@ -150,80 +150,78 @@ else if (temp.parent.data < temp.data) {
...
-            // If the successor has a right child, the child's grandparent is it's new parent
-            if (successor.parent != temp) {
-                if (successor.right != null) {
-                    successor.right.parent = successor.parent;
-                    successor.parent.left = successor.right;
-                    successor.right = temp.right;
-                    successor.right.parent = successor;
-                } else {
-                    successor.parent.left = null;
+        else {
```
Comment (albina-astr) [inline src/main/java/com/thealgorithms/datastructures/trees/BinaryTree.java:154]: I wouldn't do that. The logic behind `if - else if - else` structure is more readable and easier to understand.

File: src/main/java/com/thealgorithms/datastructures/trees/BinaryTree.java [hunk 2]
```diff
@@ -150,80 +150,78 @@ else if (temp.parent.data < temp.data) {
...
-
-            // The left tree of temp is made the left tree of the successor
-            successor.left = temp.left;
-            successor.left.parent = successor;
-
-            // If the successor has a right child, the child's grandparent is it's new parent
-            if (successor.parent != temp) {
-                if (successor.right != null) {
-                    successor.right.parent = successor.parent;
-                    successor.parent.left = successor.right;
```
Comment (albina-astr) [inline src/main/java/com/thealgorithms/datastructures/trees/BinaryTree.java:166]: This change is OK.

**Review by albina-astr — CHANGES_REQUESTED** [review]
Thanks for your effort! I see 1 improvement that is really helpful here (duplicate lines fix). Regarding the remaining changes, I think they are redundant, as the current code looks more readable than the new version.

**Pull request closed without merging**